grafana-plugin/src/containers/RotationForm/parts/DateTimePicker.tsx

```diff
@@ -60,9 +60,12 @@ const DateTimePicker = observer(
 
     const getDateForDatePicker = () => {
       const date = new Date();
+      // Day of the month needs to be set at first to prevent incorrect month increment
```

Comment:
_nit_ maybe worthwhile adding a unit test for this case?

Reply:
Good idea, thank you! I'll add it shortly

Reply:
Done and helped to actually catch one more edge case 👍